TestCases/species_transport/venturi_primitive_3species/species2_primitiveVenturi_mixingmodel.cfg

```diff
@@ -34,6 +34,8 @@ FLUID_MODEL= FLUID_MIXTURE
 %
 MOLECULAR_WEIGHT= 28.96, 16.043
 %
+SPECIFIC_HEAT_CP = 1009.39, 1009.39
+%
```

Comment:
What is the convergence behavior when Cp_1 and Cp_2 are different? Maybe put Cp_2=2000 here to show that it is actually OK to have different Cp.